docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Plasma Troubleshooting
+summary: Troubleshooting Issues With the Plasma Desktop
+---
+
+# Plasma Desktop Troubleshooting
+
+This contains fixes for various issues you might encounter with the Plasma desktop environment.
+
+## Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
+
+This is often caused by a Firefox bug. It can be triggered by various things, like moving Firefox windows.
+
+**Symptoms:**
+- Can't click on anything on the desktop.
+- Bringing up windows with keyboard shortcuts (ex: Super for the app menu, F9 for a drop down terminal) works, but they don't get focused. You can't interact with them.
+- Mouse cursor is stuck in move mode, it looks like a cross.
+
+You may not need to reboot to recover from this state. Killing all firefox processes may be all that's necessary. To do so:
+
+- Get to a virtual terminal (TTY) with Ctrl-Alt-F3
+- Run `killall firefox`, this should close all Firefox windows
+- Return to your Plasma session with Ctrl-Alt-F2
+
+You should now be able to use Plasma normally.
+
+## Application windows look corrupt / incomplete
+
+**Symptoms**
+- Application windows look corrupt. For example, NeoChat not displaying room or user icons.
+- Menus in an application are blank.
+
+You may be able to fix this by clearing KDE / Plasma cache files.
+NOTE: You should not be logged into your Plasma session while doing this. It is also highly recommended to back up the folders mentioned below before deleting anything.
```

Comment:
Since this is part of the procedure, there's no need to add a note about it.

Reply:
I converted this to a warning. I think it's important to emphasize this because if a user skips steps, thinking they can do this from the regular Plasma session, they need to know it will probably cause them problems. This may also cause us to have questions in the support channels.